moe._999eagle.OkunaDesktop.appdata.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2019 Sophie Tauchert -->
+<component type="desktop-application">
+	<id>moe._999eagle.OkunaDesktop</id>
```

Comment:
why the underscore here? the domain is just https://999eagle.moe/ right?

Reply:
Flatpak builder didn't accept a domain component starting with a number. I though this was the best way to do it.

Reply:
oh interesting, I wonder if that's an inherited dbus requirement or something else. I'll have a quick dig. But yeah, that's a legit answer.

Reply:
Yeah, that's dbus thing.